### DISCUSSION TITLE
Add text search filters on each column in the Table view

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#16723] [Feature Requests]
Related to #16702 and #10517

From the perspective of an end user, this is the most basic and needed search functionality when dealing with table rows imo. This search would not be based upon the database values directly but instead upon the text values being displayed in the table, because those are the values that the user cares about. The user wouldn't need to understand the relational database model at all in order to search for a collection item based on a value in a relational table, and generally it would greatly improve the ease of use and search-ability of collections for non-technical users. If this feature could be implemented it would be a huge plus.

Example: 
![table-search-example](https://user-images.githubusercontent.com/41580234/205703289-169d0740-01ce-4856-9216-e3589dce4cf9.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
